### PR TITLE
Generate errors for undefined global functions.

### PR DESCRIPTION
**Pull request by mckoss**
@matjaz 

This is a variant of your PR.  I wanted to defer failing so we could output multiple errors all at once.

Let me know what you think, and I'll merge this to master.

### CODE REVIEW

File: lib/rules-generator.js [hunk 1]
```diff
@@ -366,9 +377,31 @@ methods(Generator, {
...
+    }
+    // TODO: Validate "method" calls against a prototype.
+    return undefined;
   },
 
+  silent: function(f) {
+    if (f === undefined) {
+      f = true;
+    }
+    this.runSilently = f;
```
Comment (matjaz) [inline lib/rules-generator.js]: `f !== false`

**Comment by matjaz** [comment]
LGTM

File: lib/rules-generator.js (hunk 1 above)
Comment (mckoss) [inline lib/rules-generator.js]: I didn't indent to "clean" a truthy value here - just to allow for optional argument.

**Comment by mckoss** [comment]
Thanks for the review!

**Pull request merged**